exercises/practice/dominoes/.meta/example.ml

```diff
@@ -1,4 +1,4 @@
-(* Based off the Haskell solution by Tarmean at http://exercism.io/submissions/6dc2eef7e7eb469d8657111fc4389fc0 *)
+(* Based off the Haskell solution by Tarmean at http://exercism.org/submissions/6dc2eef7e7eb469d8657111fc4389fc0 *)
```

Comment:
This link (old and new) give me a 404.

@exercism/ocaml, @iHiD, @ErikSchierboom do we want to remove this line or find the submission and fix the link?

Reply:
I've done a bit of digging, and I think this is the correct v3 link: https://exercism.org/tracks/haskell/exercises/dominoes/solutions/457a1115cdd141c0a8dea14af57c7e46